library_generation/test/integration_tests.py

```diff
@@ -73,9 +82,12 @@ def test_generate_repo(self):
                     f"with the actual library in {repo_dest}/{library_name}. "
```

Comment:
Should we change the actual library path in log?

Reply:
Indeed

Reply:
Fixed